pyproject.toml

```
@@ -52,7 +52,8 @@ python-pptx = "*"
 pylatexenc = "*"
 tornado = "*"
 python-dotenv = "*"
-protobuf = ">=4.21.6,<6.0.0" # chromadb currently fails on 5.0+
+protobuf = "^4.21.6" # chromadb currently fails on 5.0+
+opentelemetry-exporter-otlp-proto-grpc = "1.25.0"
```

Is it necessary to pin opentelemetry-api as well?
```suggestion
opentelemetry-api = "1.25.0"
opentelemetry-exporter-otlp-proto-grpc = "1.25.0"
```

---

I think just protobuf < 5.0.0 will do it. It worked for amanape. Let's just force both and we'll see how chroma solves it eventually.